cmd/bundle/init.go

```diff
@@ -0,0 +1,64 @@
+package bundle
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/databricks/cli/libs/git"
+	"github.com/databricks/cli/libs/template"
+	"github.com/spf13/cobra"
+)
+
+var gitUrlPrefixes = []string{
+	"https://",
+	"git@",
+}
+
+func newInitCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init TEMPLATE_PATH",
+		Short: "Initialize Template",
+		Args:  cobra.ExactArgs(1),
+	}
+
+	var configFile string
+	var projectDir string
+	cmd.Flags().StringVar(&configFile, "config-file", "", "Input parameters for template initialization.")
+	cmd.Flags().StringVar(&projectDir, "project-dir", "", "The project will be initialized in this directory.")
+	cmd.MarkFlagRequired("output-dir")
```

Comment:
if it's required you can make it a positional argument, but I don't have a strong preference

Reply:
This is required for now, but we may change this later. The structure of the command has not been closed on yet. The plan is to go ahead with this, and potentially change this based on feedback.

relevant design decision: https://docs.google.com/document/d/1SRH7c-nX9V37_oZWRJU3WGjye96W02XWpGsAUpvLPQs/edit#bookmark=id.tu4ypfn79pvd